core/server/master/src/main/java/alluxio/master/file/meta/InodeTreePersistentState.java

```diff
@@ -542,6 +542,8 @@ private void applyInode(MutableInode<?> inode) {
     // becomes visible at this point.
     mInodeStore.writeInode(inode);
     mInodeStore.addChild(inode.getParentId(), inode);
+    // Only update size, last modified time is updated separately.
+    updateLastModifiedAndChildCount(inode.getParentId(), Long.MIN_VALUE, 1);
```

Comment:
Could you point me to where the last mod time gets updated?

Reply:
https://github.com/Alluxio/alluxio/blob/metastore-branch/core/server/master/src/main/java/alluxio/master/file/meta/InodeTree.java#L678

Reply:
Got it, thanks